docs/release-notes/8.0.37-29.md

```diff
@@ -0,0 +1,58 @@
+# Percona XtraDB Cluster 8.0.37-29 (2024-0-0)
+
+[Get started with Quickstart Guide for Percona XtraDB Cluster.](../quickstart-overview.md)
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.37-29 (2024-0-0) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.37-29.html).
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.37 and included in Percona XtraDB Cluster are the following:
+
+This release enhances telemetry and provides a more comprehensive explanation of how the telemetry works, including new components, metrics files, and additional methods for disabling telemetry. Find more information in the [Telemetry on Percona XtraDB Cluster](../telemetry.md) document.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.37 and included in Percona Server for MySQL are the following:
+
+* The redo log might not record a change in column order when using instant DDL, potentially leading to an inaccurate log replay during the recovery process. (Bug #35183686)
+
+* Enhanced management of buffers has been implemented in deleting tablespaces to prevent a possible assertion failure. (Bug #35676106, Bug #36343647)
+
+* Resolved an assertion failure associated with full-text indexes. (Bug #35836581)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.37 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-37.html).
+
+## Bug fixes
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): Percona XtraDB Cluster node stoped when parallel replication workers were executing DDLs via async node.
+
+* [PXC-4313](https://perconadev.atlassian.net/browse/PXC-4313): Statements executed in RSU mode generated local GTID events.
+
+* [PXC-4363](https://perconadev.atlassian.net/browse/PXC-4363): Concurrent CREATE and DROP USER queries on different nodes resulted in permanent lock.
```

Comment:
CREATE USER and DROP User
resulted in "a" permanent lock.

Reply:
added